book/02-git-basics/sections/tagging.asc

```diff
@@ -57,6 +57,8 @@ Annotierte Tags werden dagegen als vollständige Objekte in der Git-Datenbank ge
 Sie werden mit einer Prüfsumme versehen, enthalten den Tagger-Namen, die E-Mail-Adresse und das Datum, haben eine Tagging-Meldung und können mit GNU Privacy Guard (GPG) signiert und überprüft werden.
 Es wird allgemein empfohlen, dass Sie annotierte Tags erstellen, damit Sie all diese Informationen erhalten können; aber wenn Sie ein temporäres Tag wünschen oder aus irgendwelchen Gründen die anderen Informationen nicht speichern wollen, sind auch nicht-annotierte Tags möglich.
 
+// Sicher, dass es im Deutschen "kommentiert" heißt?
```

Comment:
Ich bin natürlich **nicht** sicher. Ist den in der Informatik die Annotation ein eingeführter Begriff?
Dann sollte es "annotiert" heißen.

Reply:
Ich bin (auch) kein Informatiker und weiß es deshalb ebenfalls nicht. Aber wenn in dem genannten Git-Repo *nicht* von "kommentiert", sondern von "annotiert" die Rede ist, ist das schon ein ziemlich guter Hinweis sein.

Ich würde alles auf "annotiert" ändern. Machst du das (bitte!)? (Dann kann ic schon im nächsten Kapitel weiterlesen.

Reply:
Ich möchte vorher noch die anderen "Aktiven" dazu hören.
@pastatopf,  @cbachhuber,  @saxc 
Könntet ihr die Zeilen 60ff bitte mal kurz reviewen?

Reply:
Es werden in Deutschen (je nachdem mit wen man spricht) beide Begriffe verwendet. Im Zweifel würde ich immer eher zum Englischen Original Tendieren.

Reply:
Schließe mich @saxc an! Bin also hier dafür, alles auf annotiert zu ändern.

Reply:
OK, einverstanden!
Wenn dieser PR gemergt ist, erstelle ich einen neuen mit den "annoted" Anpassungen.

Oder gibt es einen Weg das hier einzubinden?

Reply:
Ein separater Commit ist glaube ich am Besten. Dann ist es auch klar, dass man das geändert hat. Man kann ja auf die Diskussion hier verweisen.

Reply:
Ist mir auch lieber so.
Dieser PR hier hat mich an manchen Stellen ziemlich verwirrt und ich musste aufpassen, dass ich an der richtigen Stelle bin.

Reply:
Wenn du einen besseren Vorschlag für PRs hast, immer raus damit. Werden ja vermutlich noch ein paar kommen ;)

Reply:
Vielleicht sollten wir pro Datei einen eigenen PR starten, die können dann ja auch "Draft" bleiben, bis sie abgeschlossen sind.

Reply:
As you wish :)